src/tsviz-app.ts

```diff
@@ -20,9 +20,8 @@ function main(args: string[]) {
 
     let dependenciesOnly = switches.indexOf("-d") >= 0 || switches.indexOf("-dependencies") >= 0; // dependencies or uml?
     let recursive = switches.indexOf("-r") >= 0 || switches.indexOf("-recursive") >= 0;
-    let svgOutput = switches.indexOf("-svg") >= 0;
```

Comment:
Don't forget to delete the switch from the console error log ... couple of lines above